lib/rubocop/cop/lint/shadowed_exception.rb

```diff
@@ -77,7 +77,21 @@ def contains_multiple_levels_of_exceptions?(group)
             return !(group.size == 2 && group.include?(NilClass))
           end
 
-          group.combination(2).any? { |a, b| a && b && a <=> b }
+          group.combination(2).any? do |a, b|
+            compare_exceptions(a, b)
+          end
+        end
+
+        def compare_exceptions(a, b)
```

Comment:
Not the biggest fan of params named `a` and `b`, even if here it's relatively clear what they mean.

Reply:
I changed the variable name in d88837e. Because the line becomes longer, I'm changing the method name a little.